src/Entity/Lifetime.php

```diff
@@ -183,4 +183,19 @@ public function getSeed(string $name, int $numberOfValues, array $weights = null
 
         return $seed->getValue();
     }
+
+    /**
+     * @param string $name
+     * @return bool
+     */
+    public function hasSeed($name)
```

Comment:
V2 branch bumped PHP requirement to >=7.1, so can use `public function hasSeed(string $name): bool` here and drop docblock as duplicate info only at that stage.